aiohttp/connector.py

```diff
@@ -791,6 +812,11 @@ def _close_immediately(self) -> List["asyncio.Future[None]"]:
             ev.cancel()
         return super()._close_immediately()
 
+    @cached_property
+    def allowed_protocol_schema_set(self) -> FrozenSet[str]:
+        """Return allowed protocol schema set."""
+        return super().allowed_protocol_schema_set | TCP_PROTOCOL_SCHEMA_SET
+
```

Comment:
Personally, I'd have kept it to a simple:
```
allowed_protocol_schema_set = HIGH_LEVEL_SCHEMA_SET.union("tcp")
```

I'm convinced on the single use globals or cached_property.

Reply:
It did end up more complex than needed.

adjusted in 374821b2f847e250e7f2e69874401ad6e80c8c1a